### PR TITLE
Added talk 05-19

### PR DESCRIPTION
**Pull request by velith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ No date is currently known, we will update Readme once we know more.
 
 By adding your checkbox, you are approving [DevOps Malmö Meetup group guidelines for speakers](guidelines.md).
 
+- [X] *2021-05-19* - Lessons Learned - Short story about measuring DevOps performance [Fredrin Stål](talks/fredrik-metrics.md)
```
Comment (gangsta) [inline README.md]: should be `Fredrin Stål -> Fredrik Stål`

**Review by gangsta — CHANGES_REQUESTED** [review]

**Review by kvarak — APPROVED** [review]

**Comment by kvarak** [comment]
It would be good if you interactively rebased this branch before merging to get a nicer history :)

**Comment by velith** [comment]
You mean squash?

**Review by gangsta — APPROVED** [review]
LGTM

**Pull request merged**